.github/workflows/ci.yml

```
@@ -15,7 +15,7 @@ jobs:
 
     strategy:
       matrix:
-        python-version: [3.8]
+        python-version: [3.9]
```

```suggestion
        python-version: [3.8, 3.9]
```
We want to support both versions.